.idea/misc.xml

```diff
@@ -7,7 +7,7 @@
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="ES6" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="mcreator_sdk" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8 (3)" project-jdk-type="JavaSDK">
```

Comment:
Files like this can be overlooked sometimes but this shouldn't be changed, do not include idea file changes when creating PR's

Reply:
Agreed, revert this file to original state.

Reply:
I'm new to Intellij so how would I go about changing this name as going into the file will and changing it there will mess up my workspace. Idk if you can even help me as I am on a Mac.